src/main/java/internetshop/controllers/AddProductController.java

```diff
@@ -0,0 +1,35 @@
+package internetshop.controllers;
+
+import internetshop.lib.Inject;
+import internetshop.lib.Injector;
+import internetshop.model.Product;
+import internetshop.service.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddProductController extends HttpServlet {
+    @Inject
+    private static final Injector injector =
```

Comment:
```suggestion
    private static final Injector INJECTOR =
```
аналогічно в інших контролерах